### PR TITLE
adding popular tags tile

### PR DESCRIPTION
**Pull request by Guntrisoft**
- added it to the tag matrix as 3rd tile

### CODE REVIEW

**Comment by Guntrisoft** [comment]
@benbarnett @jongilbraith

File: app/assets/stylesheets/_settings.scss [hunk 1]
```diff
@@ -55,6 +55,7 @@ $article-tile-bg--one: $blue-chill;
 $article-tile-bg--two: $white;
 $article-tile-bg--three: $astral;
 $article-tile-bg--mas: $atlantis;
+$article-tile-bg--popular: $cream-can;
```
Comment (benbarnett) [inline app/assets/stylesheets/_settings.scss]: mmm

File: app/assets/stylesheets/components/tiles/_article_tile.scss [hunk 2]
```diff
@@ -189,3 +202,43 @@
...
+  list-style: none;
+  padding: 0;
+  margin-top: baseline-unit(9);
+}
+
+.article-tile-list__item {
+  @include font-size(30);
+  line-height: baseline-unit(6);
+  display: inline;
+  color: #000;
```
Comment (benbarnett) [inline app/assets/stylesheets/components/tiles/_article_tile.scss]: We have a variable https://github.com/moneyadviceservice/publify/blob/popular_tags/app/assets/stylesheets/lib/_colors.scss#L4

File: app/assets/stylesheets/components/tiles/_article_tile.scss [hunk 3]
```diff
@@ -189,3 +202,43 @@
...
+
+  &:before {
+    @include font-size(25);
+    display: inline-block;
+    content: "•";
+    color: #fff;
```
Comment (benbarnett) [inline app/assets/stylesheets/components/tiles/_article_tile.scss]: Same here https://github.com/moneyadviceservice/publify/blob/popular_tags/app/assets/stylesheets/lib/_colors.scss#L2

**Comment by benbarnett** [comment]
:+1: once colour variables used

**Comment by darokel** [comment]
:+1:

**Comment by jongilbraith** [comment]
Have added the changes to use real tag counts for the popular tags.

@darokel want to give my query a once over and confirm if you reckon it's decent?

**Pull request merged**